### DISCUSSION TITLE
Encryption of SEB file to be shared with Students with password to run

### DISCUSSION THREAD

**Discussion by manish0228** [SafeExamBrowser/seb-win-refactoring#705] [Q&A]
Dear Friends,

We are conducting our online exam using Safe Exam browser. We need to share the configured SEB file with the students to run Safe exam browser with our configuration only. We set a token number in the SEB file which match with our Server and only then one can access the exam. 
We got to know that some of the students are find a link https://e-baca.com/seb-decryptor/ where they can decrypt the SEB file and breech the security.

Is there any way to prevent students from decrypting the SEB file?

Thanks 
Ravi

**Comment by dbuechel** [comment]
You'll need to set a strong settings password for your configurations, see also https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane. Once the students know the settings password, you'd obviously need to change it for the next exam.

**Reply by dbuechel** [reply]
If you'd like to avoid this scenario and furthermore benefit from a lot of exam- and security-related functionality, we'd strongly recommend the usage of SEB-Server.

**Reply by dbuechel** [reply]
And if using SEB-Server is not possible in your case, you should at least use the Configuration Key and/or Browser Exam Key, see https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane.
